engine/src/test/java/io/camunda/zeebe/engine/state/migration/DbMigratorImplTest.java

```diff
@@ -86,26 +87,79 @@ void shouldRunMigrationsInOrder() {
     inOrder.verify(mockMigration2).runMigration(mockProcessingState);
   }
 
-  private static final class RunInTransactionContext implements TransactionContext {
+  @Test
```

Comment:
🔧 Can we also have an actual state test that verifies that the version is written to the state?